dictionaryfunction.py

```diff
@@ -46,7 +46,8 @@
     "Guardians of the Galaxy": 2014,
     "Avengers: Endgame": 2019, 
     "Avengers: Infinity War": 2018, 
-    "Captain Marvel": 2019
+    "Captain Marvel": 2019, 
+    "Son Goku": 2019

```

Comment:
Please put son goku in the right place. He is an actor.

Reply:
I will make a change. What is Goku's date of birth?

Reply:
12 October 737